Microsoft.DotNet.Interactive.CSharp/CSharpKernel.cs

```diff
@@ -38,6 +42,12 @@ public class CSharpKernel :
 
         private WorkspaceFixture _fixture;
 
+        private readonly JsonSerializer _jsonSerializer = JsonSerializer.CreateDefault(new JsonSerializerSettings()
+        {
+            ContractResolver = new CamelCasePropertyNamesContractResolver(),
```

Comment:
is this convention going to be shared with all kernels doing lsp?

Reply:
Probably, but for now only `CSharpKernel.cs` does anything with LSP.  I'll move this up a level.

Reply:
Done.